src/components/pages/weather/elements/WeatherWindCell.js

```diff
@@ -2,17 +2,21 @@ import React, { Component } from 'react';
 
 export default class WeatherWindCell extends Component {
   render() {
-    const averageWind = this.props.dayWindData.avg;
-    const maxWind = this.props.dayWindData.max;
-    const minWind = this.props.dayWindData.min;
-    const directionWind = this.props.dayWindData.dir;
-    return (
+    const averageWind = (this.props.dayWindData.avg ? (this.props.dayWindData.avg) : ('data unavailable'));
+    const maxWind = (this.props.dayWindData.max ? (this.props.dayWindData.max) : ('data unavailable'));
+    const minWind = (this.props.dayWindData.min ? (this.props.dayWindData.min) : ('data unavailable'));
+    const directionWind = (this.props.dayWindData.dir ? (this.props.dayWindData.dir) : ('data unavailable'));
+    return ((!this.props.dayWindData.avg && !this.props.dayWindData.max && !this.props.dayWindData.min) ? (
+      <div className="wind-cell">
+        <p>no wind data available</p>
+      </div>
+    ) : (
       <div className="wind-cell">
         <h4>Wind Speed: {averageWind}</h4>
         <p>Max: {maxWind}</p>
         <p>Min: {minWind}</p>
         <p>Direction: {directionWind}</p>
       </div>
-    );
+    ));
   }
```

Comment:
Similarly in this file in regards to:

- Using the **OR** operator (`||`)
- Trying to avoid the **NOT** operator (`!`)
- Thinking about which parentheses are necessary in our ternary operator.